VERSION

```diff
@@ -1 +1 @@
-v1.7.0-dev
+v0.1.0-dev
```

Comment:
these changes make a lot of sense, but I'd create separate PRs for those

Reply:
Thanks! I remove it from this PR.